rhel8/profiles/ism_o.profile

```diff
@@ -0,0 +1,145 @@
+  Information Security Manual can be found at the ACSC website:
+
+  https://www.cyber.gov.au/ism
+
+extends: e8
+
+selections:
+
+  ## Operating system configuration
+  ## Identifiers 1491
+  - no_shelllogin_for_systemaccounts
+
+  ## Local administrator accounts
+  ## Identifiers 1410
+  - accounts_password_all_shadowed
+
+  ## Content filtering & Anti virus
+  ## Identifiers 1341 / 1034 / 1417 / 1288
+  - package_aide_installed
+
+  ## Software firewall
+  ## Identifiers 1416
+  - configure_firewalld_ports
+  - configure_firewalld_rate_limiting
+  - firewalld_sshd_port_enabled
+  - set_firewalld_default_zone
+
+  ## Endpoint device control software
+  ## Identifiers 1418
+  - package_usbguard_installed
+  - service_usbguard_enabled
+  - usbguard_rules_not_empty_not_missing
+
+  ## Authentication hardening
+  ## Identifiers 1546 / 0974 / 1173 / 1504 / 1505 / 1401 / 1559 / 1560
+  ## 1561 / 0421 / 1557 / 0422 / 1558 / 1403 / 0431
+  - disable_host_auth
+  - require_emergency_target_auth
+  - require_singleuser_auth
+  - sebool_authlogin_nsswitch_use_ldap
+  - sebool_authlogin_radius
+  - sshd_disable_kerb_auth
+  - sshd_set_max_auth_tries
+  - sssd_enable_smartcards
+  - accounts_password_minlen_login_defs
+  - var_password_pam_minlen=14
+  - accounts_password_pam_minlen
+  - accounts_password_pam_minclass
+  - accounts_password_pam_dcredit
+  - accounts_password_pam_lcredit
+  - accounts_password_pam_ocredit
+  - accounts_password_pam_ucredit
+  - accounts_password_pam_maxrepeat
+  - accounts_passwords_pam_faillock_deny
+  - accounts_passwords_pam_faillock_deny_root
+  - accounts_passwords_pam_faillock_interval
+  - accounts_passwords_pam_faillock_unlock_time
+
+  ## Password authentication & Protecting credentials
+  ## Identifiers 1055 / 0418 / 1402
+  - network_nmcli_permissions
+  - configure_kerberos_crypto_policy
+  - kerberos_disable_no_keytab
+  - sebool_kerberos_enabled
+  - sshd_disable_gssapi_auth
+  - enable_ldap_client
+  - set_password_hashing_algorithm_libuserconf
+  - set_password_hashing_algorithm_logindefs
+  - set_password_hashing_algorithm_systemauth
+  - accounts_password_warn_age_login_defs
+  - accounts_maximum_age_login_defs
+  - accounts_minimum_age_login_defs
+ 
+  ## System administration & MFA
+  ## Identifiers 1382 / 1384 / 1386
+  - package_sudo_installed
+  - configure_opensc_card_drivers
+  - force_opensc_card_drivers
+  - sebool_authlogin_yubikey
+
+  ## System patching & Applicatoin versions
+  ## Identifiers 1493 / 1144 / 0940 / 1472 / 1494 / 1495 / 1467 / 1483 
+  - dnf-automatic_apply_updates
+  - package_dnf-plugin-subscription-manager_installed
+  - package_subscription-manager_installed
+
+  ## Centralised logging facility
+  ## Identifiers 1405 / 0988 
+  - rsyslog_cron_logging
+  - rsyslog_files_groupownership
+  - rsyslog_files_ownership
+  - rsyslog_files_permissions
+  - rsyslog_nolisten
+  - rsyslog_remote_loghost
+  - rsyslog_remote_tls
+  - rsyslog_remote_tls_cacert
+  - service_ntpd_enabled
```

Comment:
Supported ntp software on rhel8 is Chrony afaik. Could you change selections? We have some rules for Chrony, see the services/ntp folder.

Reply:
Thank you @vojtapolasek. Swapped out ntpd for Chrony https://github.com/wcushen/content/commit/d713d44d7afd256afd201fc0c3e80cad1f09ea5b